modules/vpc-endpoints/main.tf

```diff
@@ -0,0 +1,53 @@
+locals {
+  enabled = var.context.enabled && (length(var.gateway_vpc_endpoints) > 0 || length(var.interface_vpc_endpoints) > 0)
+}
+
+data "aws_vpc" "vpc" {
+  count = local.enabled ? 1 : 0
+  id    = var.vpc_id
+}
+
+data "aws_vpc_endpoint_service" "gateway_endpoint_service" {
+  for_each     = var.gateway_vpc_endpoints
+  service      = var.gateway_vpc_endpoints[each.key].name
+  service_type = "Gateway"
+}
+
+data "aws_vpc_endpoint_service" "interface_endpoint_service" {
+  for_each     = var.interface_vpc_endpoints
+  service      = var.interface_vpc_endpoints[each.key].name
+  service_type = "Interface"
+}
+
+module "label" {
```

Comment:
same here, we don't need another label if we don't override anything in the existing `module.this` label